src/components/workspace/Tabs/Tabs.tsx

```diff
@@ -11,6 +17,31 @@ interface IRenameFile {
   newPath: string;
 }
 
+type ContextMenuKeys = 'close' | 'closeOthers' | 'closeAll';
+
+interface IContextMenuItems extends MenuProps {
+  key: ContextMenuKeys;
+  items: {
```

Comment:
Nested objects should almost always be defined as a separate type alias. Necessity of `IContextMenuItems['items']` supports this.